pages/content/amp-dev/documentation/index.html

```diff
@@ -0,0 +1,221 @@
+---
+$title: Get Started
+$titles:
+  navigation: Get started
+$view: /views/custom.j2
+$path: /documentation/index.html
+$localization:
+  path: /{locale}/documentation/index.html
+$order: 1
+$hidden: true
+flyout:
+  description: Get started
+  icon: start
+
+format_cards:
+  - title: AMP Websites
+    description: Create flawless websites. Compelling, smooth and instant.
+    type: websites
+    link:
+        text: Build your first AMP Website
+        url: /content/amp-dev/documentation/guides-and-tutorials/start/create/?format=websites
```

Comment:
Something seems wrong here: we shouldn't mix GET parameters (`?format=websites`) with Grow references. I am even surprised this doesn't break 😅